app/views/track.js

```diff
@@ -131,9 +176,49 @@ const TrackView = Mn.LayoutView.extend({
         });
     },
 
+    shouldDisplay() {
```

Comment:
Puting a `display:none` on each unwanted tracks is much consuming rendering process in CSSOM. I may suggest you take a look at https://github.com/cozy/cozy-contacts/blob/master/client/app/routes/tags.coffee#L12-L18 that generate a stylesheet ine the document where we can dynamically add / remove rules to display / hide the CSS class concerned. It'l be more efficient as just one rendering loop is involved to to the hide trick.